extension/test/e2e/e2e-test-utils.js

```diff
@@ -24,7 +24,7 @@ async function executeInAdyenIframe (page, selector, executeFn) {
 
 async function initPuppeteerBrowser () {
   return puppeteer.launch({
-    headless: true,
+    headless: false,
```

Comment:
it should be headless :D

Reply:
Oh damn, good catch 😄

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/335/commits/faa5b3d639e628e02ef7dedcf84ee55f5ff9a09e